test/common/config/subscription_factory_test.cc

```diff
@@ -190,18 +188,26 @@ INSTANTIATE_TEST_CASE_P(SubscriptionFactoryTestApiConfigSource,
                                           envoy::api::v2::core::ApiConfigSource::REST,
                                           envoy::api::v2::core::ApiConfigSource::GRPC));
 
+// TODO(jbuckland) add test case for ApiConfigSource::GRPC
```

Comment:
Oops, I'll remove this. I had at one point removed `envoy::api::v2::core::ApiConfigSource::GRPC` from the values block above.